pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -175,14 +196,20 @@ func (s *spdxParser) getFileElement(elementID string) []*model.PkgInputSpec {
 
 func (s *spdxParser) GetPredicates(ctx context.Context) *assembler.IngestPredicates {
 	logger := logging.FromContext(ctx)
-
 	preds := &assembler.IngestPredicates{}
 
-	toplevel := s.getPackageElement("DOCUMENT")
+	topLevel := s.getPackageElement("DOCUMENT")
 	// adding top level package edge manually for all depends on package
-	if toplevel != nil {
-		preds.IsDependency = append(preds.IsDependency, common.CreateTopLevelIsDeps(toplevel[0], s.packagePackages, s.filePackages, "top-level package GUAC heuristic connecting to each file/package")...)
-		preds.HasSBOM = append(preds.HasSBOM, common.CreateTopLevelHasSBOM(toplevel[0], s.doc))
+	if topLevel != nil {
```

Comment:
nit; i think this technically should never be nil now, so maybe we can throw an error if it is

Reply:
This method doesn't return an error so I'll leave it